### ISSUE TITLE
Request: Swap A/B and X/Y controller icons

### ISSUE THREAD

**Issue by joshbirnholz** [Lyall/MetaphorFix#70]
The controller icons in this game for the Nintendo Switch Pro Controller ("Type 4" in the game menu) have the A/B and X/Y icons swapped.

You can see in the screenshot below that the positioning of the A/B and X/Y buttons are swapped from where they are on the controller. Instead it is using the same layout as on Xbox controllers.

![Screenshot 2024-10-15 100606](https://github.com/user-attachments/assets/dffae4bc-994c-4eda-87b7-76c5b64a3cde)

Not only is the positioning wrong, but the icons correspond to the wrong mapping with the default controls. For instance, when the game asks you to press X, you need to press Y. To solve this at the moment, the controls would have to be re-mapped in Steam so that the game prompts are correct, AND then the game controls would need to be rebound in-game so that the positioning is correct in battle.

I have a setup where sometimes I stream the game with Moonlight and use a Switch controller and sometimes I play at my desktop with a PS5 controller. I would have to re-bind all the controls every time I use a different controller in order to get the accurate prompts in-game.

Long story short, this could all be more easily fixed if the game just had the A/B and X/Y icons swapped for Switch controllers (and Switch controller only)! Is this something that could be fixed by this mod?

**Comment by Hadi10000** [comment]
I think the issue you have is with steam if you go to Steam>Stettings>Controller
![image](https://github.com/user-attachments/assets/be98a812-64a1-4ba4-b741-1ece60040514)
you should see if this option is enabled or not.

**Comment by joshbirnholz** [comment]
> I think the issue you have is with steam if you go to Steam>Stettings>Controller ![image](https://private-user-images.githubusercontent.com/35529403/376708715-be98a812-64a1-4ba4-b741-1ece60040514.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkwMTYzNTIsIm5iZiI6MTcyOTAxNjA1MiwicGF0aCI6Ii8zNTUyOTQwMy8zNzY3MDg3MTUtYmU5OGE4MTItNjRhMS00YmE0LWI3NDEtMWVjZTYwMDQwNTE0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE1VDE4MTQxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIxYmQ0NWFmODIxY2I5OTRkNjlkOThjODE3MzljMTEwOTY4MjYwNWU1YjdlNTA0MDE1NmZkYzg5MTBiOGZiODcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.znPWIqhEoE_dr33wB3CDcNDyss_5qNHzzH16AsLB8k4) you should see if this option is enabled or not.

That unfortunately doesn't solve the issue. Enabling that option makes the game prompts correct (ie, when it asks you to press X, you actually can press X instead of having to press Y) but the layout is still wrong unless you rebind the controls in the game's settings.

If you were only going to play one type of controller, that would be fine, but since I use a Switch controller when streaming and a PlayStation controller when at my desk, it would be annoying to rebind the controls every time.

**Comment by 13xforever** [comment]
You can remap the buttons in game settings, and it will swap in game.

**Comment by Romangelo** [comment]
This is the same issue as Persona 5 on Steam. They're too lazy to make alternate battle UI for people who swapped confirm and cancel buttons.

To fix this, you'll have to mod the UI textures. I did it for P5 with Special-K, but I think it'd be better if someone can directly mod the game files instead.
